services/bower/bower.tester.js

```diff
@@ -56,21 +65,30 @@ t.create('custom label for pre version') // e.g. pre version|v0.2.5-alpha-rc-pre
 
 t.create('Version for Invaild Package')
   .get('/v/it-is-a-invalid-package-should-error.json')
-  .expectJSON({ name: 'bower', value: 'invalid' })
+  .expectJSON({ name: 'bower', value: 'not found' })
 
 t.create('Pre Version for Invaild Package')
   .get('/vpre/it-is-a-invalid-package-should-error.json')
-  .expectJSON({ name: 'bower', value: 'invalid' })
+  .expectJSON({ name: 'bower', value: 'not found' })
 
 t.create('licence for Invaild Package')
   .get('/l/it-is-a-invalid-package-should-error.json')
-  .expectJSON({ name: 'license', value: 'invalid' })
+  .expectJSON({ name: 'license', value: 'not found' })
 
-t.create('Version label should be `no releases` if no official version')
+t.create('Version label should be `no releases` if no stable version')
   .get('/v/bootstrap.json')
   .intercept(nock =>
     nock('https://libraries.io')
       .get('/api/bower/bootstrap')
-      .reply(200, { latest_stable_release: { name: null } })
```

Comment:
I'm not 100% convinced `{ latest_stable_release: { name: null } }` is a real condition. I think you actually get `{ latest_stable_release: null }` . I suspect this was reverse-engineered from the code when the tests were written rather than based on a real API response. I've said in the schema that if `latest_stable_release` is an object, `name` is required. We'll render `invalid response data` if this does actually happen.